src/main/java/net/minecraftforge/client/gui/widget/ScrollPanel.java

```diff
@@ -242,7 +242,7 @@ public boolean mouseClicked(double mouseX, double mouseY, int button)
         if (super.mouseClicked(mouseX, mouseY, button))
             return true;
 
-        this.scrolling = button == 0 && mouseX >= barLeft && mouseX < barLeft + barWidth;
+        this.scrolling = button == 0 && mouseX >= barLeft && mouseX < barLeft + barWidth && mouseY >= top && mouseY <= bottom;
```

Comment:
I will try to get a chance to test this somewhat soon as maybe it is just due to me having never used the ScrollPanel and not knowing what it uses for various values... But looking at it the fact `mouseX` is compared using `< barLeft + barWidth`, but `mouseY` is compared using `<= bottom` feels like one of the two checks is probably off by a pixel due to the `<` vs `<=`.